### PR TITLE
[testing]Test new pipeline

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: eng/pipelines/link-verification.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - /*.md
```
Comment (weshaggard) [inline eng/pipelines/link-verification.yml:9]: Did you try just `*.md`?

Comment (sima-zhu) [reply eng/pipelines/link-verification.yml:9]: Yes, we need to have single quotes around this in yaml. It doesn't work as expected.

**Pull request closed without merging**